.vscode/settings.json

```
@@ -0,0 +1,22 @@
+{
+  "window.title": "${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}[${activeRepositoryBranchName}]",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  // To use this settings please install `sysoev.vscode-open-in-github` extension
```

should we just put these at the end of the file?

```suggestion
  // For `sysoev.vscode-open-in-github` extension
```